eq_cir_management_ui/config/config.py

```diff
@@ -0,0 +1,36 @@
+import os
```

Comment:
Worth adding an `__init__.py` in the same folder as this file too

Reply:
added